CHANGELOG.md

```diff
@@ -1,5 +1,106 @@
 # Changelog
 
+## [v1.14.0](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.14.0) (2023-12-19)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.13.1...v1.14.0)
+
+**Breaking changes:**
+
+- fix!: replaces .String\(\) method with internal method .TimeToString\(\) to align formating with Atlas API. [\#1699](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1699) ([marcosuma](https://github.com/marcosuma))
+   - Fixes date fields that were not compliant to the ISO 8601 timestamp format in UTC, in line with the documentation.
+- feat!: new required attributes `authorized_user_first_name` and `authorized_user_last_name` in `mongodbatlas_backup_compliance_policy` resource. [\#1655](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1655) ([marcosuma](https://github.com/marcosuma))
+   - Adds first and last name in `mongodbatlas_backup_compliance_policy` resource to reflect recent changes in the Atlas API.
+
+**Enhancements**
+
+- feat: new `mongodbatlas_stream_instance` resource [\#1685](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1685) ([AgustinBettati](https://github.com/AgustinBettati))
+- feat: new `mongodbatlas_stream_instance` and `mongodbatlas_stream_instances` data sources [\#1689](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1689) [\#1701](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1701) ([AgustinBettati](https://github.com/AgustinBettati))
+- feat: new `mongodbatlas_stream_connection` resource [\#1736](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1736) ([AgustinBettati](https://github.com/AgustinBettati))
+- feat: new `mongodbatlas_stream_connection` and `mongodbatlas_stream_connections` data sources [\#1757](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1757) ([AgustinBettati](https://github.com/AgustinBettati))
+
+**Bug Fixes**
+
+- fix: Doesn't disable X.509 in the project when `mongodbatlas_x509_authentication_database_user` resource is deleted [\#1760](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1760) ([lantoli](https://github.com/lantoli))
+- fix: adjust root `project_id` attribute in `mongdbatlas_project_api_key` resource to optional [\#1664](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1664) ([AgustinBettati](https://github.com/AgustinBettati))
+- fix: define `project_assignment` block in `mongodbatlas_project_api_key` as required to avoid plugin crash [\#1663](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1663) ([AgustinBettati](https://github.com/AgustinBettati))
```

Comment:
```suggestion
- fix: Defines `project_assignment` block as required in `mongodbatlas_project_api_key`  to avoid plugin crash [\#1663](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1663) ([AgustinBettati](https://github.com/AgustinBettati))
```